src/_includes/feature_lists/self-hosted.njk

```diff
@@ -0,0 +1,106 @@
+{%
+    set sections = [{
+        label: "Features",
+        hosting: "self-hosted",
+        rows: [{
+            id: "team-members",
+            label: "Team Members",
+            values: ['2', '20', 'Unlimited'],
```

Comment:
```suggestion
            values: ['50', 'Unlimited', 'Unlimited'],
```

Reply:
@MarianRaphael Not sure here, can you expand on your reasoning?